client/package.json

```diff
@@ -7,6 +7,7 @@
     "jquery": "^3.3.1",
     "popper.js": "^1.14.7",
     "react": "^16.7.0",
+    "react-bootstrap": "^1.0.0-beta.5",
```

Comment:
Bootstrap 4 is already packaged at the moment, and I believe react-bootstrap uses Bootstrap 3, so we may need to pick one or the other moving forward to avoid weird Bootstrap bugs.

Reply:
I'm okay with just using bootstrap 4. I imported react-bootstrap to format the login page a bit but ended up just writing some css since it's just a form.